### PR TITLE
P90 single connection latency graphs

### PR DESCRIPTION
**Pull request by csujedihy**
This is what it looks like and it's in both summary and detailed view. I've deleted the existing non-per-commit latency graph.
image

And some minor clean-up that speeds up the graph generation for a few seconds and makes it more readable.

### CODE REVIEW

File: detailed.html [hunk 1]
```diff
@@ -416,7 +416,7 @@ <h1 class="h3 mb-0 font-weight-bold text-gray-800">Performance Detailed View</h1
                             <div class="card shadow mb-4">
                                 <!-- Card Header -->
                                 <div class="card-header py-3 d-flex flex-row align-items-center justify-content-between">
-                                    <a class="m-0 font-weight-bold text-dark" href="detailed.html#rpslatency">Requests per Second Latency (μs)</a>
+                                    <a class="m-0 font-weight-bold text-dark" href="detailed.html#rpslatency">Requests per Second P90 Latency (μs)</a>
```
Comment (nibanks) [inline detailed.html:419]: I want to keep RPS. I want to add latency.

File: assets/summary/main/data.js [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
... 6160689, 6165306]}, {c:106, m:"09", b:0, d:[6093158, 6138780, 6137332, 6169102, 4027335]}, {c:105, m:"09", b:0, d:[5973309, 5954871, 5905843, 5943090, 5953020]}, {c:104, m:"09", b:0, d:[4617824, 5080219, 5685666, 5286722, 5080834]}, {c:103, m:"09", b:0, d:[6116752, 6093645, 6124104, 6145181, 4937814]}, {c:102, m:"09", b:0, d:[5878099, 6178338, 6178014, 5899289, 5982426]}, {c:101, m:"09", b:0, d:[5867950, 5970222, 6071885, 5985293, 5990734]}, {c:100, m:"09", b:0, d:[6050860, 6161896, 6128312, 6127274, 6141968]}, {c:99, m:"09", b:0, d:[6062245, 3935156, 6150900, 4565760, 6171656]}, {c:98, m:"09", b:0, d:[5946993, 5930222, 5009563, 6137515, 5972007]}, {c:97, m:"09", b:0, d:[6024271, 6121540, 6132105, 6124894, 6116804]}, {c:96, m:"09", b:0, d:[6052201, 6111222, 5795971, 5871675, 6105714]}, {c:95, m:"09", b:0, d:[5798160, 5954986, 5947590, 5943315, 5949700]}, {c:94, m:"09", b:0, d:[5902523, 6025527, 6031303, 6032529, 5999191]}, {c:93, m:"09", b:0, d:[5924815, 6038310, 6029263, 6059476, 6049318]}, {c:92, m:"09", b:0, d:[6033291, 6064807, 6159408, 6149825, 6140872]}, {c:91, m:"09", b:0, d:[5853804, 5908820, 6155202, 6139578, 5647697]}, {c:90, m:"09", b:0, d:[5993090, 6125447, 6145528, 61
```
Comment (nibanks) [inline assets/summary/main/data.js:35]: Make sure to merge/pull the latest and regenerate, so you don't lose recent commits.

File: detailed.html (hunk 1 above)
Comment (csujedihy) [reply detailed.html:419]: I am not removing RPS. I am replacing RPS latency with per-commit P90 RPS latency. The original RPS latency graph which is only for the latest commit doesn't seem very useful. I can add P99, P999 and P50 if you want, which is basically (sort of) a super set of the original latency graph. What do you think?

File: assets/summary/rpsLatencyCharts.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+                    beforeBody: beforeBodyPlacement
+                }
+            }
+        }
+    });
+
+    if (toggleEnabled) {
+        platformTypes.forEach(x => addPlatformToggle(test, x, chart))
+    }
+}
```
Comment (nibanks) [inline assets/summary/rpsLatencyCharts.js:96]: nit: add a trailing newline

File: detailed.html (hunk 1 above)
Comment (nibanks) [reply detailed.html:419]: I still want to be able to see the latest-commit RPS latency. I do find it useful to see how the curves compare between the plaftorms.

Comment (csujedihy) [reply detailed.html:419]: Ok, I will add it back.

Comment (csujedihy) [reply detailed.html:419]: Done

**Review by nibanks — APPROVED** [review]
LGTM, but please keep an eye on the next couple of CI runs to make sure things go smoothly.

**Pull request merged**